index.html

```diff
@@ -2003,6 +2009,70 @@ <h3>Agility vs. Hardening</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Transformations</h3>
+
+        <p>
+At times, it is beneficial to transform the data being protected during the
+cryptographic protection process. Such "in-line" transformation can enable a particular type
+of cryptographic protection to be agnostic to the data format it is carried in.
+For example, some Data Integrity cryptographic suites utilize RDF Dataset
+Canonicalization [[?URDNA2015]] which transforms the initial representation into a
+canonical form [[?N-QUADS]] that is then serialized, hashed, and digitally
+signed. As long as any syntax expressing the protected data can be transformed
+into this canonical form, the digital signature can be verified. This enables
+the same digital signature over the information to be expressed in JSON, CBOR,
+YAML, and other compatible syntaxes without having to recreate the cryptographic
+proof for every syntax.
+        </p>
+        <p>
+Being able to express the same digital signature across a variety of syntaxes is
+beneficial because systems often have native data formats with which they
+operate. For example, some systems are written against JSON data, while others
+are written against CBOR data. Without transformation, systems that process
+their data internally as CBOR are required to store the digitally signed data
+structures as JSON (or vice-versa). This leads to double-storing data and can
+lead to extra security attack surface if the unsigned representation stored in
+databases accidentally deviates from the signed representation. By using
+transformations, the digital proof can live in the native data format to
+help prevent otherwise undetectable database drift over time.
+        </p>
+        <p>
+This specification is designed to avoid requiring the duplication of signed
+information by utilizing "in-line" data transformations. Application developers are urged
+to work with cryptographically protected data in the native data format for
+their application and not separate storage of cryptographic proofs from the data
+being protected. Developers are also urged to regularly confirm that the
+cryptographically protected data has not been tampered with as it is written to
+and read from application storage.
+        </p>
+        <p class="issue"
+          title="Collision-resistant canonicalization requirements">
+The VCWG is seeking feedback on normative language that cryptographic suite
+implementers need to follow to ensure that they do not utilize data
+transformation mechanisms that can map to the same output. That is, given
+the different inputs for canonicalization scheme #1 and canonicalization
```

Comment:
```suggestion
different inputs for canonicalization scheme #1 and canonicalization
```

Reply:
Both are logically equivalent statements. I think you are just stating the logically polar opposite of what is in the spec right now? I think these are the two statements under evaluation:

1. Pushing different inputs through two canonicalization schemes MUST NOT produce the same output value. This is what is in the spec text, if this happens, you have successfully produced a collision.
2. Pushing the same input through two canonicalization schemes MUST produce different output values. This is what I think you're saying. If this happens, you have successfully produced a collision.

I've been staring at this now for 10 minutes and am not 100% sure that I haven't confused myself.